### PR TITLE
[WIP] Animations

### PR DESCRIPTION
**Pull request by jkeussen**

### CODE REVIEW

File: src/css/lib/animations.css [hunk 1]
```diff
@@ -1,10 +1,15 @@
 .animated {
-  -webkit-animation-duration: 0.5s;
-  animation-duration: 0.5s;
+  -webkit-animation-duration: 1s;
+  animation-duration: 1s;
```
Comment (adamschroder) [inline src/css/lib/animations.css]: i'd like to see each of these broken out into separate classes. duration-1,2 etc

Comment (iamtimm) [inline src/css/lib/animations.css]: I also think you don't need the browser prefix here. Not entirely sure though.

Comment (netroy) [inline src/css/lib/animations.css]: unfortunately we do :( .. safari, up until 8, needs `-webkit-` .. safari is the new IE6

Comment (adamschroder) [inline src/css/lib/animations.css]: we should write prefixing functions then, so we don't have to repeat that garbage.

Comment (jkeussen) [inline src/css/lib/animations.css]: Thanks for the feedback!
I've created a few more classes with different durations. Do you want them to be in seconds or be "fast", "normal", "slow" (like jQuery's animations)?

Comment (adamschroder) [inline src/css/lib/animations.css]: I would be fine with word descriptors, that way we can ensure consistency for how people use it across projects. Anyone else have an opinion here?

Comment (benjaminmateev) [inline src/css/lib/animations.css]: I like fast, normal and slow more as well. Shouldnt there be build methods that can take care of prefixing automatically when building the css files?

Comment (iamtimm) [inline src/css/lib/animations.css]: @netroy @adamschroder basscss uses cssnext as a preprocessor and that has an auto prefix feature.

**Pull request merged**